common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -286,13 +302,29 @@ namespace com.keyman.keyboards {
 
     }
 
+    static sanitize(rawRow: LayoutRow) {
+      for(const key of rawRow.key) {
+        // Test for a trailing comma included in spec, added as null object by IE
```

Comment:
Is this an issue now that we no longer support IE?

Reply:
That's a decent question.  It's old enough code that I'm not sure if it _only_ occurred in IE or not, and it's relatively harmless as-is, I think... so I didn't want to risk breaking anything.

Reply:
Sounds Good To Me